app/controllers/my_notifications.js

```diff
@@ -0,0 +1,223 @@
+import Controller from "@ember/controller";
+import _ from "lodash";
+import { later } from "@ember/runloop";
+import { inject as service } from "@ember/service";
+import EmberObject, { computed } from "@ember/object";
+
+const MSG_KEY = msg => {
```

Comment:
`MSG_KEY`
this is a convention of a constant, rather than a function.
Probably you can change it to `getMessageKey`